src/main/java/org/project36/qualopt/domain/Participant.java

```diff
@@ -44,8 +48,19 @@
     @Column(name = "number_of_repositories")
     private Integer numberOfRepositories;
 
+    @Column(name = "has_opted_in")
+    private boolean hasOptedIn = true;
+
+    @Column(name = "number_of_invitations_requested")
+    private Integer numberOfInvitationsRequested = DEFAULT_NUMBER_OF_INVITATIONS;
+
+    @Column(name = "incentive_choice")
+    private String incentiveChoice = DEFAULT_INCENTIVE_CHOICE; //Currently a string, need to change to an enum when Issue 5 is pulled in
```

Comment:
Potentially change this to the enum now that #5 has been integrated?

Reply:
Yea potentially in another pull request. Due to the size and nature of this one, pulling this in would be ideal to help others deal with conflicts well in advance

Reply:
makes sense!